src/sentry/api/endpoints/internal/integration_proxy.py

```diff
@@ -183,6 +185,13 @@ def http_method_not_allowed(self, request):
         self.log_extra["host"] = request.headers.get("Host")
 
         if not self._should_operate(request):
+            self.log_extra["silo_mode"] = SiloMode.get_current_mode().value
+            logger.info(
+                "integration_proxy.bad_request",
+                extra={
+                    **self.log_extra,
+                },
```

Comment:
do you need to do **? Looks like everywhere else we're just using the log_extra dict

Reply:
oh i copied it from when i was passing in some extra stuff